File: JavaCoreGoIt/src/module02/practice20160820/Task02.java
package module02.practice20160820;


class Task02 {

    private static int countN(int[] arr, int n) {
        int q=0;
        for ( int i: arr ) if (i == n) q++;
        return q;
    }
    public static void main(String[] args) {
        int[] array = {13, 24,46,989,42, 42, 53};
        int n = 42;

        int result=countN(array,n);
        if (result==1)  System.out.println("yes");
        else if (result==0)  System.out.println("no");
        else if (result>1)  System.out.println(result);


    }

}

File: JavaCoreGoIt/z_kotlin/src/module02/practice20160820/Task02.kt
package module02.practice20160820

internal fun countN(arr: IntArray, n: Int): Int {
    var q = 0
    for (i in arr) if (i == n) q++
    return q
}

fun main(args: Array<String>) {
    val array = intArrayOf(13, 24, 46, 989, 42, 42, 53)
    val n = 42
    val result = countN(array, n)

    when (result) {
        0 -> println("no")
        1 -> println("yes")
        else -> println(result)
    }

}